### ISSUE TITLE
Documentation in contributing is out of date with respect to FSAC

### ISSUE THREAD

**Issue by 7sharp9** [ionide/ionide-vscode-fsharp#1219]
**Is your feature request related to a problem? Please describe.**
contributing document is out of date so following instructions will not work

**Describe the solution you'd like**
Documentation correct

**Describe alternatives you've considered**
guess

**Additional context**
Section in question is:
```
Working with FSAC

Run FSAC:
or from sources in your favorite editor: open the FsAutoComplete.sln, start debugging with .NET Core mode http (debug) configuration (default to port 8088)
or run it with --mode http --port 8088 arguments
In vscode, set the FSharp.fsacUrl setting to http://localhost:8088
Reload the instance with > Developer: Reload Window.
The port number can be changed as needed.

Remove that setting to go back to FSAC bundled in Ionide extension.
```
`FSharp.fsacUrl ` is no longer valid

**Comment by ZaymonFC** [comment]
Possibly related #1197

**Comment by 7sharp9** [comment]
Fixed via #1267
